Source/OnlineSubsystemEpic/Private/OnlinePrensenceEpic.cpp

```diff
@@ -1,9 +1,11 @@
 #include "OnlinePrensenceEpic.h"
 #include "eos_presence.h"
 #include "OnlineSubsystemEpicTypes.h"
+
```

Comment:
Please don't add additional whitespace and newlines to existing code.

Reply:
Yea, my b I'll be more cautious about that going forward